spinalcordtoolbox/scripts/sct_label_vertebrae.py

```diff
@@ -15,7 +15,7 @@
 from spinalcordtoolbox.image import Image, generate_output_file
 from spinalcordtoolbox.vertebrae.core import (
     get_z_and_disc_values_from_label, vertebral_detection, expand_labels,
-    crop_labels, label_vert, EmptyArrayError)
+    crop_labels, label_vert, EmptyArrayError, MissingDiscsError)
```

Comment:
Huh. I didn't realize there was already an existing `EmptyArrayError`! Since we're checking `.getNonZeroCoordinates()` (i.e. this would be an empty array), could we just re-use the existing error type? 

And, perhaps I could use the same error for https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4199, too?

Reply:
Well, `EmptyArrayError` is used for another specific situation:
https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/21333c5c7322985e4f54ae6f6f337347462d2479/spinalcordtoolbox/vertebrae/core.py#L259-L269

I think re-using it would defeat the purpose of knowing exactly who threw each custom exception. And I can't really imagine a scenario where a caller would want to deal with an empty disc file _or_ trying to compute the center of mass of an empty array in the same way. They feel like different exceptions to me.